desktop_version/src/Screen.cpp

```diff
@@ -184,11 +205,9 @@ void Screen::FlipScreen()
 	SDL_FillRect(m_screen, NULL, 0x00000000);
 }
 
-int Screen::toggleFullScreen()
+void Screen::toggleFullScreen()
 {
 	isWindowed = !isWindowed;
-	int result = ResizeScreen(-1, -1);
```

Comment:
We still need the resize, right? Other than that lgtm

Reply:
Haha, whoops, looks like I got a little too delete-happy.

Reply:
Ok, fixed now.